OJP/OJP_JourneySupport.xsd

```diff
@@ -664,14 +657,29 @@
 			</xs:element>
 			<xs:element name="NoRamp" type="xs:boolean" default="false" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>The user is not able to use an ramp.</xs:documentation>
+					<xs:documentation>The user is not able to use a ramp.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="NoTravelator" type="xs:boolean" default="false" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The user is not able to use a travelator.</xs:documentation>
```

Comment:
Just ecstatically. Can we move this one below NoSight?

Reply:
done